netket/optimizer/sr/sr_treemv_logic.py

```diff
@@ -41,12 +41,12 @@ def f(p, x):
 
 @partial(jax.vmap, in_axes=(None, None, 0))
 def perex_grads_rc(forward_fn, params, samples):
```

Comment:
Is this called perex just because we used to call something perex in the past?
I never got that name...
If you want to pick a better name the honour is yours.

Reply:
Its called perex_grads because it was used in the old netket, I think it comes from one of the jax tutorials...

do you have a suggestion?

Reply:
Yes i know. I personally never liked it. I don't think it tells what this really is
But if you think it is, that's fine by me.
This is like a vmapped grad, no?

Reply:
The equivalent function in the matrix-based code is just called `gradients`. I think that's descriptive enough given the file context

Reply:
perex -> per-example

But what it does is calculate the jacobain in a more efficient way

Reply:
I am also in favour of renaming, it took me a while to find what perex means...

Reply:
vmap_grad with a note that it's a more efficient implementation?